app/controllers/equipment_models_controller.rb

```diff
@@ -122,6 +137,7 @@ def deactivate
         r.archive(current_user, 'The equipment model was deactivated.')
          .save(validate: false)
       end
+      OrderingHelper.new(@equipment_model).deactivate_order
```

Comment:
can we try putting this after the call to `super` so we can just check if the model has been successfully deactivated before updating the ordering? 

this may eliminate the need for `OrderingRepairer`.